composer.json

```diff
@@ -11,10 +11,11 @@
     ],
     "require" : {
         "php": ">=5.4",
+        "swiftmailer/swiftmailer": "^5.1",
         "tijsverkoyen/css-to-inline-styles": "^1.5"
     },
     "require-dev" : {
-        "swiftmailer/swiftmailer": "^5.1"
+        "phpunit/phpunit": "^4.0"
```

Comment:
@hkdobrev If unit tests are compatible with 5.x, I don't see any problem to use the latest stable.

If you are ok, I may update the pull request.